homebrew/Formula/php-pie.rb

```diff
@@ -0,0 +1,17 @@
+class PhpPie < Formula
+  desc "PIE - the PHP Installer for Extensions"
+  homepage "https://github.com/php/pie"
+  url "https://github.com/php/pie/releases/download/0.2.0/pie.phar"
```

Comment:
We should not hard-code this, the release process should be automated, and this will **definitely** be forgotten if it is not automated :)

Reply:
Personally I went with making a separate repo for this, which would be expected to be written on by a bothttps://github.com/box-project/homebrew-box

Reply:
Could be https://github.com/php/pie/releases/latest/download/pie.phar